### PR TITLE
Migrate from Travis CI to GitHub CI

### PR DESCRIPTION
**Pull request by dagwieers**
This PR includes:
- Migrate from Travis CI to GitHub CI
- Move the tests from test/ to the standard tests/ directory
- Exclude non-essential files from export
- Move codecov.yml to the only remaining location
- Add Kodi's add-on checker to GitHub CI

Also:
- Fixes an add-checker issue in addon.xml
- Fixes a translation issue with `fr_fr`

For this PR the tests on Matrix and Python 3.8 have been disabled
because of issues, resp. with missing dependencies on Matrix and issues
with the code.

### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
GitHub CI has quite some improvements over Travis CI.

### CODE REVIEW

**Comment by dagwieers** [comment]
You can see the results from GitHub CI here: https://github.com/dagwieers/plugin.video.netflix/actions

Unfortunately we don't do run tests and coverage currently, but we can improve on this later. Also SonarCloud support is easy to add as well. We have those provisions in other projects.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Kodi version](https://img.shields.io/badge/kodi%20versions-18-blue)](https://kodi.tv/)
 [![GitHub release](https://img.shields.io/github/release/castagnait/plugin.video.netflix.svg)](https://github.com/castagnait/plugin.video.netflix/releases)
-[![Build Status](https://travis-ci.org/castagnait/plugin.video.netflix.svg?branch=master)](https://travis-ci.org/castagnait/plugin.video.netflix)
+[![CI](https://github.com/castagnait/plugin.video.netflix/workflows/CI/badge.svg)](https://github.com/castagnait/plugin.video.netflix/actions?query=workflow:CI)
```
Comment (CastagnaIT) [inline README.md]: This have a weird path i don't have these addresses, previous ones have always worked

Comment (dagwieers) [reply README.md]: It doesn't work because it is not merged yet. As soon as this is merged the URL will start working. See inputstreamhelper.

Comment (dagwieers) [reply README.md]: Or the link in my comment above.

**Comment by dagwieers** [comment]
In the VRT NU project we even us GitHub CI for periodically testing authentication and other services to detect API changes or subtle breakage proactively (i.e. before users report it).

**Comment by CastagnaIT** [comment]
now understand
but i am a bit confused..
with this PR merged, on website `https://travis-ci.org/` is have to disable it?
are the same builds or different builds?

**Comment by dagwieers** [comment]
We no longer use Travis CI after this PR, but instead migrate to using GitHub CI. An integrated CI service from GitHub which works much better nowadays. As a plus we now also run kodi-addon-chdcker (for Leia only as Matrix has dependencies missing).

**Pull request merged**